src/main/kotlin/racing/domain/validation/CarRaceValidation.kt

```diff
@@ -0,0 +1,16 @@
+package racing.domain.validation
+
+import racing.domain.exception.CarNameLengthExcessException
+
+class CarRaceValidation {
+
+    fun validateCarName(carNames: List<String>) {
+        if (carNames.none { CAR_NAME_RANGE.contains(it.trim().length) }) {
+            throw CarNameLengthExcessException("자동차 이름은 ${CAR_NAME_RANGE.first}자 이상이어야 하고, ${CAR_NAME_RANGE.last}자를 초과할 수 없습니다.")
+        }
+    }
+
+    companion object {
+        private val CAR_NAME_RANGE = IntRange(1, 5)
```

Comment:
```suggestion
        private val CAR_NAME_RANGE = 1..5
```
range를 상수로 처리해주셔서 range 인스턴스가 한번만 생성되게 되었습니다 👍  참고로 다음과 같이 작성할 수도 있어요!

Reply:
반영했습니다-!

Reply:
꿀팁 감사합니다. 반영했습니다-!